packit/constants.py

```diff
@@ -38,3 +38,25 @@
 GH2FED_RELEASE_TOPIC = "org.fedoraproject.prod.github.release"
 
 DEFAULT_BODHI_NOTE = "New upstream release: {version}"
+
+PACKIT_CONFIG_TEMPLATE = """# This configuration file is generated. See the documentation for more information:
+# https://github.com/packit-service/packit/blob/master/docs/configuration.md
+
+specfile_path: {{ downstream_package_name }}.spec
+
+# add or remove files that should be synced
+synced_files:
+    - {{ downstream_package_name }}.spec
+
+# change these lines if upstream project name is not the same as downstream
+upstream_project_name: {{ upstream_project_name }}
+downstream_package_name: {{ downstream_package_name }}
+
+# These values are optional and filled with defaults.
```

Comment:
I debate if this section is a good idea: whenever we add/change something in the config file, we will have to update it.

I'd personally start with a small concise config.